_posts/2022-09-10-executive-director-pyopensci-leah-wasser-updates.md

```diff
@@ -0,0 +1,123 @@
+  * Once tools are accepted into our ecosystem, we will track tool maintenance and find new maintainers, or support sunsetting as it makes sense to ensure users aren't left hanging.
+  * Through peer review we will also address redundancy in tool functionality across the scientific python ecosystem.
+* Create a model to support peer review within domain specific subcommunities. We will begin our work here with the Pangeo community. Here, we will support a peer review process with community-specific standards applied on top of ours to ensure tool consistency.   
+* Provide training and mentorship in skills needed to contribute to open source. These skills are not just for open source. They are also fundamental to good open science practices and will support career trajectories in open science too. 
+* Grow support for maintainers. A critical component of this effort will be a communication campaign around the importance of open source to science. 
+
+### We will also continue to curate our partnership with the Journal of Open Source Software (JOSS). 
+Package maintainers can still submit a package to us if they wish to publish with 
+JOSS. Our partnership supports a "fast track" review in JOSS if its functionality 
+fits their scope and requirements. Once a package is accepted by pyOpenSci , if 
+the package fits into the JOSS scope our review will count towards JOSS. A package 
+author then only needs to write a paper for JOSS, and they will be "dual"
+published with us and JOSS. This, friends, is a win-win!
+
+## Who does pyOpenSci serve?
+pyOpenSci supports maintainers of smaller scientific python open source projects. 
+We also support scientists using these tools and will empower folks like you 
+with the skills needed to both contribute to Open Source and improve your open 
+science workflows! In the long term these skills will be invaluable in your 
+data intensive career trajectory!
+
+## pyOpenSci is about collaboration - not reinventing the wheel
+Collaborations with existing efforts are critical. I welcome any feedback from 
+you regarding groups that we should be engaging with! I will also need help 
+finding reviewers and editors, mentors to support training efforts and voices to 
+spread the word about the project.
+
+## And… what's the timeline?
+Unless I figure out how to create numerous robot clones of myself, please know 
+that all of the above won’t happen immediately. I am currently the sole 
+employee of the organization and this is my second week. But never fear - we 
+will accomplish a lot in the next two years and look forward to and will greatly
+value support from some of you along the way!
+
+## Stay in touch!
+There are many ways to stay in touch with our project. Once I have a road map 
+for activities developed, I will communicate this here on our 
+[website](https://www.pyopensci.org/), Twitter 
+and [discourse](https://pyopensci.discourse.group/). Discourse is full open
```

Comment:
"Discourse is fully open"

Reply:
you can totally edit the text inline in the future, @NickleDave if you wish using inline suggestions. then i can just accept them!! but for now i'm editing locally.

Reply:
whoops, sorry.

I thought there was a way to edit inline but I couldn't figure it out 🙁 
Will do so next time